tests/context.py

```diff
@@ -135,7 +135,7 @@ def get_gcd_lcm_examples(cmd, func, func_eval, multi_func):
         (f"{cmd}(6128, 24)", func(6128, 24)),
         (f"{cmd}(24, 6128)", func(24, 6128)),
         (f"{cmd}(1E20, 1000000)", func(Rational('1E20'), 1000000)),
-        (f"{cmd}(128*10^32, 1)", func(Rational('128E32'), 1)),
+        (f"{cmd}(128*10^{{32}}, 1)", func(Rational('128E32'), 1)),
```

Comment:
@builtbywill why does this need to be in double brackets? shouldn't {32} be enough? :confused:

Reply:
@lily514 because the "f" makes it a string format, so the "cmd" can be inserted, and we need to escape the braces inside